docs/content/Tooltip.mdx

```diff
@@ -10,8 +10,14 @@ import DeprecationBanner from '../components/DeprecationBanner'
 
 <DeprecationBanner replacementUrl={'/components/tooltip/react/latest'} />
 
+## Deprecation
```

Comment:
Just asking because I always get confused, should we call this "Upcoming deprecation" instead? So that it does not get confused for a breaking change while Tooltip v1 is still supported from the main bundle.

(in the next major, we would move Tooltip v1 to /deprecated bundle completing the deprecation?)

Reply:
I think it depends on if we have the next thing ready, or not. If there is something that we want people to move to, we should say something is deprecated and point them to the new thing. Otherwise I think saying it will be deprecated will work.

This gets a little complicated with our setup, though, with different components moving around 😓 I think that if we treat this process as a way to flag to people if they are okay using something at that moment then it may be the most helpful with what we have in place.